backend/middleware/sanitize.js

```
@@ -116,6 +116,48 @@ export class ValidationError extends Error {
 export function sanitizeWithSchema(data, schema, path = '') {
   // Handle null/undefined
   if (data === null || data === undefined) return data;
+  // Special case: /refresh-token strict validation
+  if (
+    schema &&
+    typeof schema === 'object' &&
+    Object.keys(schema).includes('refreshToken')
+  ) {
+    const val = data && data.refreshToken;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)